avm/res/document-db/database-account/main.bicep

```diff
@@ -176,29 +185,6 @@ param networkRestrictions networkRestrictionType = {
 @description('Optional. Setting that indicates the minimum allowed TLS version. Azure Cosmos DB for MongoDB RU and Apache Cassandra only work with TLS 1.2 or later. Defaults to "Tls12" (TLS 1.2).')
 param minimumTlsVersion string = 'Tls12'
 
-@description('Optional. Flag to indicate enabling/disabling of Burst Capacity feature on the account. Cannot be enabled for serverless accounts.')
-param enableBurstCapacity bool = true
-
-@description('Optional. Enables the cassandra connector on the Cosmos DB C* account.')
-param enableCassandraConnector bool = false
-
-@description('Optional. Flag to enable/disable the \'Partition Merge\' feature on the account.')
-param enablePartitionMerge bool = false
-
-@description('Optional. Flag to enable/disable the \'PerRegionPerPartitionAutoscale\' feature on the account.')
-param enablePerRegionPerPartitionAutoscale bool = false
-
-@description('Optional. Analytical storage specific properties.')
-param analyticalStorageConfiguration resourceInput<'Microsoft.DocumentDB/databaseAccounts@2025-04-15'>.properties.analyticalStorageConfiguration?
-
-@description('Optional. The CORS policy for the Cosmos DB database account.')
-param cors resourceInput<'Microsoft.DocumentDB/databaseAccounts@2025-04-15'>.properties.cors?
-
-@description('Optional. The default identity for accessing key vault used in features like customer managed keys. Use `FirstPartyIdentity` to use the tenant-level CosmosDB enterprise application. The default identity needs to be explicitly set by the users.')
-param defaultIdentity defaultIdentityType = {
-  name: 'FirstPartyIdentity'
-}
```

Comment:
Beware, all these parameters were removed. It appears in your attempt to resolve conflicts you may have overwritten all other contributions